packages/ui5-middleware-index/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "ui5-middleware-index",
+  "version": "0.0.1",
+  "description": "UI5 middleware for delivering a dedicated welcome/start page",
+  "author": "Volker Buzek, Peter Muessig",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/petermuessig/ui5-ecosystem-showcase.git"
+  },
+  "dependencies": {
+    "@ui5/logger": "^2.0.0"
```

Comment:
OK - maybe we can use the 1.x version of the logger to make it usable also with the old tooling. After seeing this I am thinking of whether we should upgrade the middlewares to the tooling 2.0 or we only do it for the application?!